gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -533,4 +533,12 @@ ChangeRequest getChangeRequest(String zoneName, String changeRequestId,
    * @see <a href="https://cloud.google.com/dns/api/v1/changes/list">Cloud DNS Chages: list</a>
    */
   Page<ChangeRequest> listChangeRequests(String zoneName, ChangeRequestListOption... options);
+
+  /**
+   * Initiates a new empty batch ready to be populated with service calls, which will use this
+   * {@code Dns} instance when submitted for processing to Google Cloud DNS.
```

Comment:
Nit (feel free to ignore this): I think elsewhere I used "to issue the request", instead of "for processing to Google Cloud DNS."